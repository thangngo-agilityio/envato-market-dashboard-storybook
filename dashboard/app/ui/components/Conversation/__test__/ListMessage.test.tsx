import ListMessages from '../ListMessage';

const { render } = testLibReactUtils;

describe('ListMessages render', () => {
  test('Should render match with snapshot.', () => {
    const { container } = render(
      <ListMessages
        messages={[
          {
            date: {
              nanoseconds: 1,
              seconds: 12,
            },
            id: '12',
            senderId: '1',
            text: 'Hello',
          },
        ]}
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
