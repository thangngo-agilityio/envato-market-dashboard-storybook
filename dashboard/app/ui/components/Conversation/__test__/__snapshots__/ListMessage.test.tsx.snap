// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListMessages render Should render match with snapshot. 1`] = `
<div>
  <div
    class="css-h5aeco"
  >
    <div
      class="css-1guw43s"
    >
      <div
        class="css-14cwlhy"
      >
        <div
          class="css-7w350g"
        >
          <img
            alt="/images/avatar-user-chat.webp"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=3840&q=75"
            srcset="/_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=640&q=75 640w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=750&q=75 750w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=828&q=75 828w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Favatar-user-chat.webp&w=3840&q=75 3840w"
            style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; border-radius: 50%; margin-right: 8px; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
          />
        </div>
        <div
          class="css-u4p24i"
        >
          <div
            class="css-1sl4v5d"
            data-testid="image-container"
          >
            Hello
          </div>
          <div
            class="css-17xejub"
          />
          <p
            class="chakra-text css-t1z24v"
          >
            07:00 AM
          </p>
        </div>
      </div>
    </div>
    <form
      class="chakra-stack css-tl3ftk"
    >
      <div
        class="css-1nlrkd1"
      >
        <div
          class="css-u4p24i"
        >
          <div
            class="css-14ethi5"
          />
        </div>
        <p
          class="chakra-text css-rj1t1b"
        />
        <button
          class="chakra-button css-9xp0iw"
          disabled=""
          type="submit"
        >
          <span
            class="chakra-button__icon css-1wh2kri"
          >
            <svg
              fill="none"
              height="20"
              viewBox="0 0 20 20"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15.0586 7.09154L7.92522 3.52487C3.13355 1.12487 1.16689 3.09153 3.56689 7.8832L4.29189 9.3332C4.50022 9.7582 4.50022 10.2499 4.29189 10.6749L3.56689 12.1165C1.16689 16.9082 3.12522 18.8749 7.92522 16.4749L15.0586 12.9082C18.2586 11.3082 18.2586 8.69153 15.0586 7.09154ZM12.3669 10.6249H7.86689C7.52522 10.6249 7.24189 10.3415 7.24189 9.99987C7.24189 9.6582 7.52522 9.37487 7.86689 9.37487H12.3669C12.7086 9.37487 12.9919 9.6582 12.9919 9.99987C12.9919 10.3415 12.7086 10.6249 12.3669 10.6249Z"
                fill="white"
              />
            </svg>
          </span>
          Send
        </button>
      </div>
    </form>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
